daffodil-core/src/main/scala/org/apache/daffodil/core/dpath/Expression.scala

```diff
@@ -2113,7 +2113,7 @@ case class FunctionCallExpression(functionQNameString: String, expressions: List
           functionQName,
           args,
           NodeInfo.Long,
-          NodeInfo.Exists,
+          NodeInfo.Scalar,
           NodeInfo.String,
           DFDLContentLength(_),
         )
```

Comment:
Please find the comment on the next line (not shown in the diff) and update it so it no longer refers to `Exists` but refers to `Scalar` instead.

Reply:
Pretty sure you don't need Scalar. Just use Exists.

Exists means "I don't care about the type, I just need it to be a PathExpression"

More about this below in other comments.